<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="AKHILESG-IN" name="Test" tests="5" failures="3" timestamp="15 Mar 2020 09:31:33 GMT" time="3.069" errors="0">
  <testcase name="desting3" time="1.017" classname="TestNGDemo.NewTest">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.desting3() didn&amp;apos;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.desting3() didn't finish within the time-out 1000
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 3 stack frames]]>
    </failure>
  </testcase> <!-- desting3 -->
  <testcase name="desting3" time="1.005" classname="TestNGDemo.NewTest">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.desting3() didn&amp;apos;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.desting3() didn't finish within the time-out 1000
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 2 stack frames]]>
    </failure>
  </testcase> <!-- desting3 -->
  <testcase name="desting3" time="1.002" classname="TestNGDemo.NewTest">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method org.testng.internal.TestNGMethod.desting3() didn&amp;apos;t finish within the time-out 1000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method org.testng.internal.TestNGMethod.desting3() didn't finish within the time-out 1000
	at java.lang.Thread.currentThread(Native Method)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 2 stack frames]]>
    </failure>
  </testcase> <!-- desting3 -->
  <testcase name="testing3" time="0.003" classname="TestNGDemo.NewTest"/>
  <testcase name="doHomePageTest" time="0.004" classname="TestNGDemo.LearnTestNG"/>
</testsuite> <!-- Test -->
